docs/contributing.md

```diff
@@ -28,16 +28,15 @@ Since Flarum is so extension-driven, we highly recommend [our extension docs](ex
 
 ### Setting Up a Local Codebase
 
-[flarum/flarum](https://github.com/flarum/flarum) is a "skeleton" application which uses Composer to download [flarum/core](https://github.com/flarum/core) and a [bunch of extensions](https://github.com/flarum). In order to work on these, we recommend forking and cloning them into a [Composer path repository](https://getcomposer.org/doc/05-repositories.md#path):
+[flarum/flarum](https://github.com/flarum/flarum) is a "skeleton" application which uses Composer to download the core package and a bunch of extensions within the monorepo [flarum/framework](https://github.com/flarum/framework). In order to work on these, we recommend forking and cloning the repository into a [Composer path repository](https://getcomposer.org/doc/05-repositories.md#path):
 
 ```bash
 git clone https://github.com/flarum/flarum.git
 cd flarum
 
-# Set up a Composer path repository for Flarum packages
-composer config repositories.0 path "packages/*"
-git clone https://github.com/<username>/core.git packages/core
-git clone https://github.com/<username>/tags.git packages/tags # etc
+# Set up a Composer path repository for Flarum monorepo packages
+composer config repositories.0 path "packages/framework/*"
```

Comment:
```suggestion
# Set up a Composer path repository for Flarum monorepo packages
composer config repositories.0 path "PATH_TO_MONOREPO/*/*"
```

Reply:
Note that the double star (/*/*) is deliberate. Should we mention that?

Reply:
Oh right, we changed the monorepo structure, I don't believe a mention is necessary, most will copy past.